pkg/ui/app/components/dropdown.tsx

```diff
@@ -20,8 +31,10 @@ export default class Selector extends React.Component<SelectorOwnProps, {}> {
   render() {
```

Comment:
This component should be called "Dropdown" everywhere.  We are mixing "Selector" and "Dropdown", and that's very confusing when combined with the third-party "Select" component we are using.